worlds/yugioh06/Options.py

```diff
@@ -0,0 +1,179 @@
+import typing
+from dataclasses import dataclass
+
+from Options import Option, Choice, Range, Toggle, PerGameCommonOptions
+
+
+class StructureDeck(Choice):
+    """Which Structure Deck you start with"""
+    display_name = "Structure Deck"
+    option_dragons_roar = 0
+    option_zombie_madness = 1
+    option_blazing_destruction = 2
+    option_fury_from_the_deep = 3
+    option_warriors_triumph = 4
+    option_spellcasters_judgement = 5
+    option_none = 6
+    option_random_deck = 7
```

Comment:
fyi Choice already has a random option built in. I'm not certain how you want to manage that along with your custom random value

Reply:
random_deck doesn't allow none to be chosen.